jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/ImmutableHttpFields.java

```diff
@@ -59,7 +58,7 @@ public HttpFields asImmutable()
     public int hashCode()
     {
         int hash = 0;
```

Comment:
shouldn't the hash function start with an arbitrary prime number?

Reply:
Agreed, since we do xor's in the loop, starting with a hash of 31 would improve the spreading.

Reply:
@gregw this should be a prime number.